conans/cli/common.py

```diff
@@ -127,3 +127,10 @@ def get_lockfile(lockfile, strict=False):
         graph_lock.strict = strict
         ConanOutput().info("Using lockfile: '{}'".format(lockfile))
     return graph_lock
+
+
+def get_multiple_remotes(conan_api, remote_names=None):
```

Comment:
I know this function has been moved, from the api, but the access to ``conan_api.cache.remotes_registry.list()`` in this layer is a smell, it should be a call to the api